Client/Ecosia/Extensions/AppSettingsTableViewController+Ecosia.swift

```diff
@@ -157,9 +157,21 @@ extension AppSettingsTableViewController {
             ResetSearchCount(settings: self),
             EngagementServiceIdentifierSetting(settings: self),
             FasterInactiveTabs(settings: self, settingsDelegate: self),
-            UnleashOnboardingCardNTPSetting(settings: self),
         ]
         
+        if OnboardingCardNTPExperiment.isEnabled {
+            hiddenDebugSettings.append(UnleashOnboardingCardNTPSetting(settings: self))
+        }
```

Comment:
I don't think this condition makes sense in the case of settings that reset Unleash since we still want to be able to do it even if we end up in the control group. In those cases I think it's better to leave as it was and add it everytime.

Reply:
Make sense yeah, will revert the change, thanks!